hack/ci/e2e-k8s.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
This was intentional. Please see the commit.

Reply:
> we can decide to merge or not, I just used for convenience to avoid dealing with the timestamps 😅
https://github.com/kubernetes-sigs/kind/pull/945#issuecomment-541334267
:)

Reply:
You should add a [WIP] to the title to block merge

Reply:
yeah,my bad, sorry